test/styles/bibtex.js

```diff
@@ -0,0 +1,72 @@
+const assert = require('assert');
+const bibtexmisc = require('../../model/styles/bibtexMisc');
+const biblatexSoftware = require('../../model/styles/biblatexSoftware');
+
+const oneAuthorSourceData = require('../mockSourceData/allFieldsOneAuthor');
+const threeAuthorsSourceData = require('../mockSourceData/allFieldsThreeAuthors');
+
+describe('BibTexMiscStyle', () => {
+  describe('format', () => {
+    it('Month field, one author', () => {
+      let formattedString = bibtexmisc.format(oneAuthorSourceData);
```

Comment:
No it was not. I was under the impression that it would run automatically from our meetings, but I guess I was mistaken. I will do so later